neural_speed/models/qwen/qwen.h

```diff
@@ -44,6 +44,18 @@ static const model_scratch qwen_mem_req(int n_layers, float scratch_size_ratio =
           static_cast<unsigned long long>(scratch_size_ratio * 2048) * MB,
           static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
       };
+    case 28:
+      return {
+          static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
+          static_cast<unsigned long long>(scratch_size_ratio * 2048) * MB,
+          static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
+      };
+    case 80:
+      return {
+          static_cast<unsigned long long>(scratch_size_ratio * 10 * 4096) * MB,
```

Comment:
will times 10  compare to 28 layer is too large ?

Reply:
7b->70B,so I used 10.

Reply:
Qwen 70B have larger weight size than 7b,not only layers.